src/components/ProductForList.js

```diff
@@ -1,12 +1,23 @@
 import { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
 import { calculateEstimate } from '@the-collab-lab/shopping-list-utils/dist/calculateEstimate';
-import { updatePurchaseTimeDB } from '../lib/api';
+import { updatePurchaseTimeDB, getItemsFromList } from '../lib/api';
 import { calculateDaysSinceLastPurchase, validateHours } from '../utils/utils';
+import { deleteItems } from '../lib/api';
 
-export const ProductForList = ({ item, handleDeleteAttempt, token }) => {
+export const ProductForList = ({ item, token }) => {
   const [isBought, setIsBought] = useState(false);
 
+  const handleDeleteAttempt = async (e) => {
+    if (window.confirm('Do you want to delete this product?')) {
+      const name = e.target.getAttribute('name');
+      const itemtoDelete = await getItemsFromList(token);
+      const itemFinded = itemtoDelete.find((item) => item.name === name);
+      await deleteItems(token, itemFinded);
```

Comment:
que te parece si le agregas un salto de linea para manter seperada las cosas es decir, es un bloque los `const` y con un salto de linea el `await`  :rocket:

Reply:
got it **3a4921b16b9490999c66a48b7bf2be1e2d536666**